### PR TITLE
First viable PoC

### PR DESCRIPTION
**Pull request by Glenpl**

### CODE REVIEW

File: api_sketch/parsers.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            meta = ModelParser.parse_meta(meta)
+        if serializer_meta:
+            serializer_meta = ModelParser.parse_serializer_meta(serializer_meta)
+        return Model(model_name, fields, api_methods, app_label, endpoint_name,
+                     meta, serializer_meta)
+
+    def _extract_app_label(data):
+        try:
+            return data.split()[2]
+        except IndexError:  # it's ok - app_label is optional
```
Comment (Glenpl) [inline api_sketch/parsers.py]: Should it be optional?

Comment (Glenpl) [inline api_sketch/parsers.py]: I think it shouldnt, I'll fix it soon

**Comment by opalczynski** [comment]
@lukaszjurczak Maybe you want to contribute? ;)

**Comment by lukaszjurczak** [comment]
@opalczynski I started a new job recently and don't have enough time (that's why I didn't do your job recruitment task). For now I'm learning to understand and to review code written by others.

**Comment by opalczynski** [comment]
Feel free then :) If you change your mind - remember that this is open source now :) And all contributors are welcome!

**Pull request merged**